File: ClinicWeb/src/app/core/models/booking/clinicianBooking.model.ts
import { BookingModel } from './bookingModel.model';

export class ClinicianBookingModel extends BookingModel {
    ClinicId: number;
    ClinicName: string;
    PatientId: number;
    PatientName: string;
    PatientSurname: string;
    PatientLocation: string;
}

File: ClinicApi/Models/Booking/ClinicianBookingModel.cs
using System;

namespace ClinicApi.Models.Booking
{
    public class ClinicianBookingModel : BookingModel
    {
        public int ClinicId { get; set; }
        public string ClinicName { get; set; }
        public int PatientId { get; set; }
        public string PatientName { get; set; }
        public int PatientAge { get; set; }
    }
}